Attendee.js

```diff
@@ -0,0 +1,49 @@
+// The Attendee class can be utililized to keep certain information of the attendee, 
+// a class for the user needed to be created in addition to the session class. 
+// The test for this JS class file utilizes Jest.IO
+
+class Attendee {
+    constructor() {
```

Comment:
attendee (Java class) has three fields: session Id, screen name, and I believe date last polled